### DISCUSSION TITLE
Azure Data Studio - SELECT Results datetimeoffset format

### DISCUSSION THREAD

**Discussion by zengouu** [microsoft/azuredatastudio#24581] [Q&A]
Running a basic SELECT * FROM MyTable results show a datetimeoffset format based on my OS regional settings. For example, time separator in my country/region is a baseline dot ( . ) and Azure Data Studio query shows it as is.

This poses a problem when I'm manually copying datetimeoffset values from SELECT to a UPDATE clause, as UPDATING datetimeoffset column requires me to use colon symbol ( : ) as a separator.

Is there an OS-independent way to define how azure data studio shows the results?

Same in pictures:
What I get now:
![image](https://github.com/microsoft/azuredatastudio/assets/8158470/7695784c-eddc-47b1-b717-c9e422961d50)

What I would like to get:
![image](https://github.com/microsoft/azuredatastudio/assets/8158470/013f73ea-dd25-40ec-b840-bc0d4c3b48e7)

My workaround now is to define my OS region to be something different than what it actually is, which is hacky.

OS: MacOS Sonoma 14.0
Azure Data Studio: 1.46.1

**Comment by cheenamalhotra** [comment]
Hi @zengouu 

Have you tried exporting results to any format? Does that export results in correct format for you?
